base/rockylinux9/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM rockylinux:9
+LABEL maintainer="RStudio Docker <docker@rstudio.com>"
+
+RUN dnf -y update && \
+    dnf -y install \
+    fontconfig \
+    glibc-langpack-en \
+    perl-File-Find \
```

Comment:
Only one difference compared to the Rocky 8 image to note: TinyTeX required a different set of perl packages to install, just `perl-File-Find`, to get around this error:
```sh
Can't locate File/Find.pm in @INC (you may need to install the File::Find module) (@INC contains: /root/.TinyTeX/texmf-dist/scripts/texlive /root/.TinyTeX/tlpkg /usr/local/lib64/perl5/5.32 /usr/local/share/perl5/5.32 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5) at ./tlmgr line 86.
BEGIN failed--compilation aborted at ./tlmgr line 86.
```